frontend/EmpUI/src/features/leaves/leave-table-columns.tsx

```
@@ -44,7 +45,28 @@
       sorter: true,
       key: "employeeName",
       render: (_, record) => {
-        return <Text>{record?.employeeId}</Text>;
+          return (
+            <Link to={`/users/details/${record.employeeId}`}>
+              <Text>{record?.employeeId}</Text>;
+            </Link>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)